internal/ci/main.go

```diff
@@ -45,10 +46,19 @@ type (
 	}
 )
 
+func init() {
+	t := template.New("")
+	t.Funcs(template.FuncMap{
+		"split": strings.Split,
+		"trim":  strings.TrimSpace,
+	})
+	tpl = template.Must(t.ParseFS(tplFS, "*.tmpl"))
+}
```

Comment:
The init is not needed. See this example: https://github.com/ariga/atlas/blob/master/cmd/atlas/internal/cmdlog/cmdlog.go#L303-L316